cypress/integration/boardCRUD.spec.js

```diff
@@ -1,122 +1,123 @@
 /// <reference types="Cypress" />
-const authforms = require('../fixtures/authforms.json')
 import data from "../fixtures/data.json"
-import navigation from "../fixtures/navigation.json"
-import sidebar from "../fixtures/sidebar.json"
-import myOrganization from "../fixtures/myOrganization.json"
-import boards from "../fixtures/Boards.json"
+import authModule from "../models/authModule"
+import boards from "../models/boards"
+import organization from "../models/organization"
+import sidebar from "../models/sidebar"
+import navigation from "../models/navigations"
 
 describe('BoardCRUD', () => {
   before(() => {
-    cy.visit("/login", { timeout: 30000 })
-    cy.get(authforms.signUpForm.yourEmailInputField).clear().type(data.user.email),
-    cy.get(authforms.signUpForm.yourPasswordInputField).clear().type(data.user.password),
-    cy.get(authforms.signUpForm.submitButton).click()
-    cy.wait(3000)
+    cy.visit("/login", { timeout: 30000 }),
+    authModule.login({}),
+    cy.wait(3000),
+    cy.url().should('eq', 'https://cypress.vivifyscrum-stage.com/my-organizations'),
```

Comment:
mislim da ti je ovaj wait cist visak ovde... cy.url.should bi trebao da ceka automatski da se taj login desi.. ako ne ceka, povecaj mu timeout

Reply:
slazem se, to je zaostavština od ranije, zaboravio izbrišem 🙄